android/src/main/java/com/pw/droplet/braintree/Braintree.java

```diff
@@ -59,6 +67,41 @@ public void onPaymentMethodNonceCreated(PaymentMethodNonce paymentMethodNonce) {
           nonceCallback(paymentMethodNonce.getNonce());
         }
       });
+      this.mBraintreeFragment.addListener(new BraintreeErrorListener() {
+        @Override
+        public void onError(Exception error) {
+          if (error instanceof ErrorWithResponse) {
+            ErrorWithResponse errorWithResponse = (ErrorWithResponse) error;
+            BraintreeError cardErrors = errorWithResponse.errorFor("creditCard");
+            if (cardErrors != null) {
+              Gson gson = new Gson();
+              final Map<String, String> errors = new HashMap<>();
```

Comment:
There's no need to pull Gson in for just this, you could create a `JSONObject` here and just use `JSONObject#toString` to convert it to a string for the `nonceErrorCallback`.